<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="2" ignored="0" total="6" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:35] [INFO] com.first.utils.TestNGListener: 测试初始化</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:38] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:40] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:43] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:43] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest01' on class com.first.utils.TestngRetry Retrying 1 times</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:24:43] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:49] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:51] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:52] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:52] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:53] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:55] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:55] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest01' on class com.first.utils.TestngRetry Retrying 2 times</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:24:55] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:00] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:02] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:05] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:05] [ERROR] com.first.utils.TestNGListener: 测试用例执行失败</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:11] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:13] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:14] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest02' on class com.first.utils.TestngRetry Retrying 1 times</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:14] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:19] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:21] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:22] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest02' on class com.first.utils.TestngRetry Retrying 2 times</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:22] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:28] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:29] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
    </line>
    <line>
      <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:30] [ERROR] com.first.utils.TestNGListener: 测试用例执行失败</b></span><br />]]>
    </line>
  </reporter-output>
  <suite name="suite1" duration-ms="58365" started-at="2018-05-24T17:24:35Z" finished-at="2018-05-24T17:25:34Z">
    <groups>
    </groups>
    <test name="test1" duration-ms="58365" started-at="2018-05-24T17:24:35Z" finished-at="2018-05-24T17:25:34Z">
      <class name="com.first.testcases.LoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="3177" started-at="2018-05-24T17:24:35Z" finished-at="2018-05-24T17:24:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="loginTest01()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest01" duration-ms="5045" started-at="2018-05-24T17:24:38Z" finished-at="2018-05-24T17:24:43Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[登录测试失败 expected [true] but found [admin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 登录测试失败 expected [true] but found [admin]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at com.first.testcases.LoginTest.loginTest01(LoginTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:38] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:40] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:41] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:43] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:43] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest01' on class com.first.utils.TestngRetry Retrying 1 times</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:24:43] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest01 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3604" started-at="2018-05-24T17:24:43Z" finished-at="2018-05-24T17:24:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="2225" started-at="2018-05-24T17:24:47Z" finished-at="2018-05-24T17:24:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="loginTest01()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest01" duration-ms="5258" started-at="2018-05-24T17:24:49Z" finished-at="2018-05-24T17:24:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[登录测试失败 expected [true] but found [admin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 登录测试失败 expected [true] but found [admin]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at com.first.testcases.LoginTest.loginTest01(LoginTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:49] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:51] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:52] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:52] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:53] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:55] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:24:55] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest01' on class com.first.utils.TestngRetry Retrying 2 times</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:24:55] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest01 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3664" started-at="2018-05-24T17:24:55Z" finished-at="2018-05-24T17:24:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="2127" started-at="2018-05-24T17:24:58Z" finished-at="2018-05-24T17:25:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest01()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest01" duration-ms="4727" started-at="2018-05-24T17:25:00Z" finished-at="2018-05-24T17:25:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[登录测试失败 expected [true] but found [admin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 登录测试失败 expected [true] but found [admin]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at com.first.testcases.LoginTest.loginTest01(LoginTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:00] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:02] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 在元素 用户名输入框 输入：admin</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 在元素 密码输入框 输入：Lv931704912</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:03] [INFO] com.first.uiframework.DriverUtils: 点击元素 登录按钮</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:05] [INFO] com.first.uiframework.DriverUtils: 获取元素： 用户菜单下拉框文本值</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:05] [ERROR] com.first.utils.TestNGListener: 测试用例执行失败</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest01 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3650" started-at="2018-05-24T17:25:05Z" finished-at="2018-05-24T17:25:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="2407" started-at="2018-05-24T17:25:09Z" finished-at="2018-05-24T17:25:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="loginTest02()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest02" duration-ms="2581" started-at="2018-05-24T17:25:11Z" finished-at="2018-05-24T17:25:14Z">
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:127)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:325)
at com.first.uiframework.DriverUtils.openWeb(DriverUtils.java:29)
at com.first.pages.productpages.LoginPage.<init>(LoginPage.java:19)
at com.first.actions.LoginOperate.LoginSuess(LoginOperate.java:22)
at com.first.actions.LoginOperate.getLoginName(LoginOperate.java:30)
at com.first.testcases.LoginTest.loginTest02(LoginTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:11] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:13] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:14] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest02' on class com.first.utils.TestngRetry Retrying 1 times</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:14] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest02 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3594" started-at="2018-05-24T17:25:14Z" finished-at="2018-05-24T17:25:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="2172" started-at="2018-05-24T17:25:17Z" finished-at="2018-05-24T17:25:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="loginTest02()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest02" duration-ms="2525" started-at="2018-05-24T17:25:19Z" finished-at="2018-05-24T17:25:22Z">
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:127)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:325)
at com.first.uiframework.DriverUtils.openWeb(DriverUtils.java:29)
at com.first.pages.productpages.LoginPage.<init>(LoginPage.java:19)
at com.first.actions.LoginOperate.LoginSuess(LoginOperate.java:22)
at com.first.actions.LoginOperate.getLoginName(LoginOperate.java:30)
at com.first.testcases.LoginTest.loginTest02(LoginTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:19] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:21] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:22] [INFO] com.first.utils.TestngRetry: Running retry for 'loginTest02' on class com.first.utils.TestngRetry Retrying 2 times</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:22] [ERROR] com.first.utils.TestNGListener: 测试用例被跳过</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest02 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3629" started-at="2018-05-24T17:25:22Z" finished-at="2018-05-24T17:25:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="setup" is-config="true" duration-ms="2182" started-at="2018-05-24T17:25:26Z" finished-at="2018-05-24T17:25:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest02()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="loginTest02" duration-ms="2063" started-at="2018-05-24T17:25:28Z" finished-at="2018-05-24T17:25:30Z">
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'LAPTOP-D4Q6617H', ip: '192.168.242.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:127)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:325)
at com.first.uiframework.DriverUtils.openWeb(DriverUtils.java:29)
at com.first.pages.productpages.LoginPage.<init>(LoginPage.java:19)
at com.first.actions.LoginOperate.LoginSuess(LoginOperate.java:22)
at com.first.actions.LoginOperate.getLoginName(LoginOperate.java:30)
at com.first.testcases.LoginTest.loginTest02(LoginTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:28] [INFO] com.first.utils.TestNGListener: 测试用例开始执行</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#16A05D"><b>[2018-05-25 01:25:29] [INFO] com.first.uiframework.DriverUtils: 打开网址</b></span><br />]]>
            </line>
            <line>
              <![CDATA[<span style="color:#FF0000"><b>[2018-05-25 01:25:30] [ERROR] com.first.utils.TestNGListener: 测试用例执行失败</b></span><br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest02 -->
        <test-method status="PASS" signature="tearDowm()[pri:0, instance:com.first.testcases.LoginTest@2a798d51]" name="tearDowm" is-config="true" duration-ms="3633" started-at="2018-05-24T17:25:30Z" finished-at="2018-05-24T17:25:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
      </class> <!-- com.first.testcases.LoginTest -->
    </test> <!-- test1 -->
  </suite> <!-- suite1 -->
</testng-results>
